src/main/java/com/ebay/myriad/scheduler/TaskFactory.java

```diff
@@ -63,6 +58,31 @@ private static String getFileName(String uri) {
             }
         }
 
+        private CommandInfo getCommandInfo() {
+            MyriadExecutorConfiguration myriadExecutorConfiguration = cfg.getMyriadExecutorConfiguration();
+            String cmdPrefix = "export CAPSULE_CACHE_DIR=`pwd`;echo $CAPSULE_CACHE_DIR; java -Dcapsule.log=verbose -jar ";
+            CommandInfo.Builder commandInfo = CommandInfo.newBuilder();
+            if (myriadExecutorConfiguration.getNodeManagerUri().isPresent() &&
+                    myriadExecutorConfiguration.getCommand().isPresent() &&
+                    myriadExecutorConfiguration.isRemoteDistribution()) {
```

Comment:
Why do you even need a separate isRemoteDistribution flag? Couldn't the presence of a NodeManagerUri be enough?